### DISCUSSION TITLE
comments are almost always displayed wrong

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3625] [Mobile] [answered]
i believe i started experiencing this after the recent update

each paragraph of the comment now have a chance to be broken and show some stuff from the other paragraph (even from the other people's comments). this is severely broken and happens 100% of the time, and has already gotten me into some weird situations where i thought that people said something they actually didn't

i recorded an example with this pr https://github.com/flutter/flutter/pull/75110

https://user-images.githubusercontent.com/39104740/112762028-70b44d00-9006-11eb-87f9-5e66fbea48ec.mp4

**Comment by aconsuegra** [comment, marked as answer]
We fixed this bug in version 1.7.3. Please, update to that version of the app and let us know if the bug is still happening. Thanks!

**Reply by KalleOlaviNiemitalo** [reply]
I hit a similar bug with GitHub Mobile 1.9.0 on iOS. Viewing <https://github.com/dotnet/runtime/issues/50036> displayed a comment of mine instead of the description:

![image](https://user-images.githubusercontent.com/46201428/114305305-4ff8f680-9ae0-11eb-9f15-6a345d75dde1.png)
